### ISSUE TITLE
Camera bearing - wrong direction

### ISSUE THREAD

**Issue by tomlocks** [mapbox/mapbox-gl-native#6897]
**Platform: Nexus 5 Android 6.0.1**
**Mapbox SDK version: 4.2.0beta4**

### Steps to trigger behavior

 1. Update camera position using this code:
   ```
 final CameraPosition.Builder builder = new CameraPosition.Builder();
    builder.target(new LatLng(52.39115,16.96389)).bearing(90).zoom(12);
    final CameraUpdate cameraUpdate = CameraUpdateFactory.newCameraPosition(builder.build());
    mapboxMap.animateCamera(cameraUpdate);
```

### Expected behavior
Map should face east.
### Actual behavior
Map is facing west.

Subtracting 360 from bearing resolves this issue, but I think that it isn't correct solution.

**Comment by tobrun** [comment]
Thanks for the report, able to reproduce, fixing for the `4.2.0` release.

**Comment by tobrun** [comment]
Will be resolved when #6917 is merged in release branch, thank you for reaching out closing.

**Comment by johannesanderwald** [comment]
Hello,
When will be there a new beta with this issue fixed?
Best regards
